paymentsheet/src/main/java/com/stripe/android/paymentsheet/flowcontroller/FlowControllerViewModel.kt

```diff
@@ -27,12 +31,48 @@ internal class FlowControllerViewModel(
     // Used to determine if we need to reload the flow controller configuration.
     var previousElementsSessionParams: ElementsSessionParams? = null
 
+    val configureRequests = MutableSharedFlow<DefaultFlowController.ConfigureRequest>()
```

Comment:
Can/should we move this to it's own object, that's just referenced here in the view model (or the state component)?